package/libyui.spec

```diff
@@ -16,36 +16,48 @@
 #
 
 
-Name:           @PROJECTNAME@
-Version:        @VERSION@
+Name:           libyui
+Version:        3.2.1
 Release:        0
-Source:         @PROJECTNAME@-%{version}.tar.bz2
+Source:         libyui-%{version}.tar.bz2
 
 BuildRequires:  boost-devel
 BuildRequires:  cmake >= 2.8
 BuildRequires:  gcc-c++
 BuildRequires:  pkg-config
 
-Url:            @URL@
-Summary:        @SUMMARY@
+Url:            http://github.com/libyui/
+Summary:        GUI-abstraction library
 License:        LGPL-2.1 or LGPL-3.0
 Group:          System/Libraries
 
 %description
-@DESCRIPTION@
+This is the user interface engine that provides the abstraction from
+graphical user interfaces (Qt, Gtk) and text based user interfaces
+(ncurses).
 
-%package -n @PROJECTNAME@@SONAME_MAJOR@
+Originally developed for YaST, it can now be used independently of
+YaST for generic (C++) applications. This package has very few
+dependencies.
+
+%package -n libyui7
```

Comment:
I think this 7 should be in some macro, as it is spread over spec file, so change one value change all occurrences.